docs/user/sending-notifications.rst

```diff
@@ -18,7 +18,7 @@ Notifications can be created using the ``notify`` signal. Eg:
     notify.send(
         sender=admin,
         recipient=operators,
-        type='default',
+        type="default",
         description="Test Notification",
         verb="Test Notification",
         email_subject="Test Email Subject",
```

Comment:
are you sure this example makes perfect sense?

I was thinking we could use the generic message notification as an example which allows sending a custom message, while the example after `register_notification_type` would show how to use the typical case of sending a notification of a type which has already been defined and we don't  need to pass any message or description because it's been already defined in the type. I would like this important concept to be more easier to grasp in this document, I don't think it's clearly explained now.